src/NodaTime/LocalTime.cs

```diff
@@ -302,6 +302,36 @@ public static LocalTime FromSecondsSinceMidnight(int seconds)
             return new LocalTime(unchecked(seconds * NanosecondsPerSecond));
         }
 
+        /// <summary>
+        /// Factory method for creating a local time from the number of minutes which have elapsed since midnight.
+        /// </summary>
+        /// <param name="minutes">The number of seconds, in the range [0, 1439]</param>
```

Comment:
`The number of minutes` :)

Reply:
Humbug, I thought I was being so careful! Thanks for noticing.